data/presets/amenity/health_post.json

```diff
@@ -0,0 +1,38 @@
+{
+    "icon": "fas-house-medical-flag",
+    "fields": [
+        "name",
+        "operator",
+        "address",
+        "building_area_yes",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "healthcare",
```

Comment:
Is it feasible to create a smaller sub-set of the healthcare fields and use that here instead? Any examples I can look at?